src/textures/VideoTexture.js

```diff
@@ -1,6 +1,13 @@
 import { LinearFilter } from '../constants.js';
 import { Texture } from './Texture.js';
 
+function updateVideo() {
```

Comment:
If desired, this, instead of being a separate function, can be moved just to line 24, i.e. making something like `this._videoFrameCallback = () => { this.needsUpdate = true; this._callbackHandle = this.image.requestVideoFrameCallback( this._videoFrameCallback ); }`.